usagov/vote-gov-drupal
# Vote-768: Quicklinks Style

## Jira ticket

[Vote-768](https://cm-jira.usa.gov/browse/VOTE-768)

## Description

Adding styles for quick links section and twig template for section as well

## Deployment and testing

### Pre-deploy

1. `lando retune`

### Post-deploy

n/a

### QA/Test

1. Pull down branch and start local be sure to run `lando retune` to capture config 
2. Visit home page and verify that quick links section to match [figma design](https://www.figma.com/file/E70piAGXmi4UzrHs0A90cm/Vote.gov-Redesign-2024-(Master)?node-id=3236:15993&mode=dev) has been styled and is response with both mobile and desktop 

## Checklist for the Developer




## Checklist for the Peer Reviewers
